### ISSUE TITLE
MPL drawer displaying pi in decimal format if operated with a parameter

### ISSUE THREAD

**Issue by pedrorrivero** [Qiskit/qiskit-terra#4013]
### Information

- **Qiskit Terra version**: 0.12.0
- **Python version**: 3.7.6
- **Operating system**: MacOS Catalina 10.15.3

### What is the current behavior?

When drawing a circuit containing a gate such as `qcircuit.u3(theta, eta-pi/2, pi/2, q[0])`, the second variable is displayed as in latex `$\eta-1.5707$` instead of `$\eta-\pi/2$`.

### Steps to reproduce the problem

Plot a circuit containing `qcircuit.u3(theta, eta-pi/2, pi/2, q[0])`.

### What is the expected behavior?

The second variable should be displayed as in latex `$\eta-\pi/2$`.

**Comment by 1ucian0** [comment]
It seems like a https://github.com/Qiskit/qiskit-terra/pull/3666 needs to get deeper. What to have a look @sumitpuri?

**Comment by sumitpuri** [comment]
Sure @1ucian0 is there a bug again?

**Comment by sumitpuri** [comment]
@pedrorrivero Can you paste the code and your result.

**Comment by pedrorrivero** [comment]
Excuse my late response, I missed the notification.

```python
## IMPORTS
from qiskit import *
from qiskit.circuit import Parameter
from numpy import pi

## QUANTUM CIRCUIT
q = QuantumRegister(1, 'q')
U = QuantumCircuit(q)

# State parametrization
theta = Parameter('$θ$')
eta = Parameter('$\eta$')
U.u3(theta, eta-pi/2, pi/2, q[0])

# Draw circuit
U.draw(output='mpl')
```
![download](https://user-images.githubusercontent.com/52980156/86861241-3e377e00-c08c-11ea-9181-df7e3aaa6416.png)

**Comment by Cryoris** [comment]
This is the same as #4170.

**Comment by 1ucian0** [comment]
Fixed via #4807
